plugins/Colmena/ErrorsManager/templates/Compilations/index.php

```diff
@@ -0,0 +1,107 @@
+                                Operaciones
+                            </th><!-- .th -->
+                        <?php
+                        }
+                        ?>
+                    </tr><!-- .tr -->
+                </thead><!-- .thead -->
+                <tbody class="tbody elements">
+                    <?php
+                    foreach ($entities as $entity) {
+                    ?>
+                        <tr class="tr">
+                            <td class="td element grow">
+                                <p><?= $entity->error_id; ?></p>
+                            </td><!-- .td -->
+                            <td class="td element medium">
+                                <p><?= $entity->name?></p>
+                            </td><!-- .td -->
+                            <td class="td element grow">
+                                <p><?= $entity->message; ?></p>
+                            </td><!-- .td -->
+                            <?php
+                            if (!empty($table_buttons)) {
+                            ?>
+                                <td class="td actions">
+                                    <div class="td-content">
+                                        <?php
+                                        foreach ($table_buttons as $key => $value) {
+                                            array_push($value['url'], $entity->id);
+                                            if ($value['url']['action'] != 'delete') {
+                                                echo $this->Html->link(
+                                                    $value['icon'],
+                                                    $value['url'],
+                                                    $value['options']
+                                                );
+                                            } else {
+                                                echo $this->Form->postLink(
+                                                    $value['icon'],
+                                                    $value['url'],
+                                                    $value['options']
+                                                );
+                                            }
+                                        }
+                                        ?>
+                                    </div><!-- .td-content -->
+                                </td><!-- .td -->
+                            <?php
+                            }
+                            ?>
+                        </tr><!-- .tr -->
+                    <?php
+                    }
+                    ?>
+                </tbody><!-- .tbody -->
+            </table><!-- .table -->
+            <?= $this->element('paginator'); ?>
+        <?php
+        } else {
+        ?>
+            <p class="no-results">No existen resultados para la búsqueda realizada</p>
```

Comment:
Is there any way to encapsulate the results in a snippet?

Reply:
Do you propose to generalize the results of the templates in the same script?